src/routes/api/index.js

```diff
@@ -1,18 +1,13 @@
-const router = require("express").Router();
+import swaggerJsdoc from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+import Router from 'express';
```

Comment:
```suggestion
import { Router } from 'express';
```
Check other files for the same